docs/integrations/persisting-store-data.md

```diff
@@ -412,7 +409,7 @@ Why does it matter?
 Asynchronous hydration can cause some unexpected behaviors.
 For instance, if you use Zustand in a React app, the store will _not_ be hydrated at the initial render. In cases where you app depends on the persisted value at page load, you might want to wait until the store has been hydrated before showing anything (e.g. your app might think the user is not logged in because it's the default, while in reality the store has not been hydrated yet).
 
-If your app does depends on the persisted state at page load, see [How can I check if my store has been hydrated?](#how-can-I-check-if-my-store-has-been-hydrated) in the Q/A.
+If your app does depends on the persisted state at page load, see [How can I check if my store has been hydrated?](#how-can-i-check-if-my-store-has-been-hydrated?) in the Q/A.
```

Comment:
The inclusion of the "?" is an interesting one. For a quick bit of background, we want the docs to work on the docs site, but also, we want them to continue to work within Github. 

At the moment, the fix here makes sense in the context of the docs (which is currently broken for that link), but it wouldn't work in the context of Github, because it looks like Github strips the "?". 

I wonder if we can introduce a fix on https://github.com/pmndrs/website/ so it is consistent with github? Technically a question mark is allowed in the fragment identifier but it would be good to be consistent across both GH/docs. 

Sorry, I know that is a bit of a rabbit hole for a case fix! Is it something you want to raise/fix with the website repo? If not, I can look at doing it

Reply:
A quick update @wmcb91 , I've raised a query over on the website repo: https://github.com/pmndrs/website/issues/259
I want to get Cody's 2c on the approach. Sorry about the hold up